tests/components/homeworks/conftest.py

```diff
@@ -29,44 +74,19 @@ def mock_config_entry() -> MockConfigEntry:
         title="Lutron Homeworks",
         domain=DOMAIN,
         data={},
-        options={
-            CONF_CONTROLLER_ID: "main_controller",
-            CONF_HOST: "192.168.0.1",
-            CONF_PORT: 1234,
-            CONF_DIMMERS: [
-                {
-                    CONF_ADDR: "[02:08:01:01]",
-                    CONF_NAME: "Foyer Sconces",
-                    CONF_RATE: 1.0,
-                }
-            ],
-            CONF_KEYPADS: [
-                {
-                    CONF_ADDR: "[02:08:02:01]",
-                    CONF_NAME: "Foyer Keypad",
-                    CONF_BUTTONS: [
-                        {
-                            CONF_NAME: "Morning",
-                            CONF_NUMBER: 1,
-                            CONF_LED: True,
-                            CONF_RELEASE_DELAY: None,
-                        },
-                        {
-                            CONF_NAME: "Relax",
-                            CONF_NUMBER: 2,
-                            CONF_LED: True,
-                            CONF_RELEASE_DELAY: None,
-                        },
-                        {
-                            CONF_NAME: "Dim up",
-                            CONF_NUMBER: 3,
-                            CONF_LED: False,
-                            CONF_RELEASE_DELAY: 0.2,
-                        },
-                    ],
-                }
-            ],
-        },
+        options=CONFIG_ENTRY_DATA,
+    )
+
+
+@pytest.fixture
+def mock_config_entry_username_password() -> MockConfigEntry:
+    """Return the default mocked config entry with credentials."""
+    return MockConfigEntry(
+        title="Lutron Homeworks",
+        domain=DOMAIN,
+        data={},
+        options=CONFIG_ENTRY_DATA
+        | {CONF_PASSWORD: "hunter2", CONF_USERNAME: "username"},
```

Comment:
This data should be stored in `data`

Reply:
I don't agree. The difference between `data` and `options` is not well defined and this integration stores everything in options and nothing in data.

Reply:
We discussed this on Discord, and the conclusion is Joost is right. password + username moved to data; a follow-up PR will migrate hostname + port there too.